assets/styles/docs.scss

```diff
@@ -213,10 +213,20 @@ $bd-gutter-x: 3rem;
 
 
     .video-container {
-        padding-top: 56.25%;
         position: relative;
+        padding-top: 35.25%;
+        background-color: $black-3;
```

Comment:
Can we leave the background to match the rest of the page?

![Screenshot 2024-05-22 at 15 50 03](https://github.com/kestra-io/docs/assets/34094921/daf5d8e2-cfd4-414d-a2f0-aac9d1cf06b2)